### ISSUE TITLE
Compilation error caused by messages file

### ISSUE THREAD

**Issue by International** [smpallen99/coherence#314]
Hi everyone!

I have the following setup in mix.exs:

```
{:phoenix, "~> 1.3.0"},
{:coherence, "~> 0.5"}
```

I've ran the mix task like this:
```
mix coh.install --full --no-invitable --no-confirmable --no-recoverable --no-lockable
```

And when I try to run **mix ecto.setup**, I receive the following error:
```
== Compilation error in file lib/xtimers_web/coherence_messages.ex ==
** (ArgumentError) *gettext macros expect translation keys (msgid and msgid_plural) and
domains to expand to strings at compile-time, but the given msgid
doesn't.

Dynamic translations should be avoided as they limit gettext's
ability to extract translations from your source code. If you are
sure you need dynamic lookup, you can use the functions in the Gettext
module:

    string = "hello world"
    Gettext.gettext(XtimersWeb.Gettext, string)

    (gettext) lib/gettext/compiler.ex:213: Gettext.Compiler.expand_to_binary/4
    expanding macro: XtimersWeb.Gettext.dgettext_noop/2
    lib/xtimers_web/coherence_messages.ex:75: XtimersWeb.Coherence.Messages.you_are_using_an_invalid_security_token/0
    expanding macro: XtimersWeb.Gettext.dgettext/2
    lib/xtimers_web/coherence_messages.ex:75: XtimersWeb.Coherence.Messages.you_are_using_an_invalid_security_token/0
    (elixir) lib/kernel/parallel_compiler.ex:121: anonymous fn/4 in Kernel.ParallelCompiler.spawn_compilers/1

```

Caused by the following line in the file:

```
  def you_are_using_an_invalid_security_token,
    do: dgettext(@domain, "You are using an invalid security token for this site! This security\n" <>
      "violation has been logged.\n")
```

If I remove `<>` , I can reach **ecto.setup**.

**Comment by pups3s** [comment]
@International 
You can set `gettext` version in your deps fixed to **0.13.1** and it will work.
`[{:gettext, "0.13.1"}]`

But anyway this error must be fixed.

**Comment by schlenks** [comment]
I updated to `{:gettext, "~> 0.14"}` and still saw the error. Was only able to resolve it by removing the `<>` as suggested above.

**Comment by pups3s** [comment]
@schlenks

Not “still”.
0.14.0 is version of gettext which produces that error (with coherence).

Version 0.13.1 works correctly.
To fix version in deps you can set “0.13.1” without “~>”.

**Comment by markschultz** [comment]
is this an issue in gettext or coherence?
